build/Dockerfile

```diff
@@ -1,40 +1,63 @@
+		java-common libasan1 liblsan0 libtsan0 libubsan0 libcilkrts5 libgcc-4.9-dev \
+		libstdc++-4.9-dev libgl1-mesa-dri libgl1-mesa-glx \
+		libmono-system-xml-linq4.0-cil libmono-system-data-datasetextensions4.0-cil \
+		libstdc++-4.9-pic locales login m4 make makedev mawk mono-dmcs npm \
+		openjdk-8-jdk passwd python-distlib python-gevent python-greenlet \
+		python-html5lib python-minimal python-pip python-pkg-resources \
+		python-requests python-setuptools python-six python-urllib3 python-yaml \
+		python2.7 python2.7-minimal rpm rpm2cpio ruby ruby2.1 rubygems-integration \
+		sed tar texinfo tzdata-java udev unzip util-linux valgrind vim wget \
+		golang-go.tools curl sphinx-common gnupg python-dev python3 python3-dev \
+		elfutils &&\
+	adduser --disabled-password --gecos '' fdb &&\
+	chown -R fdb /opt &&\
+	chmod -R 0777 /opt
 
 USER fdb
 
 # wget of bintray without forcing UTF-8 encoding results in 403 Forbidden
 RUN cd /opt/ &&\
-    wget --local-encoding=UTF-8 https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2 &&\
-    echo '2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba boost_1_67_0.tar.bz2' | sha256sum -c - &&\
-    tar -xjf boost_1_67_0.tar.bz2 &&\
-    rm boost_1_67_0.tar.bz2
+  wget --local-encoding=UTF-8 https://dl.bintray.com/boostorg/release/1.67.0/source/boost_1_67_0.tar.bz2 &&\
+  echo '2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba boost_1_67_0.tar.bz2' | sha256sum -c - &&\
+  tar -xjf boost_1_67_0.tar.bz2 &&\
+  rm boost_1_67_0.tar.bz2
 
 USER root
 
-RUN pip install boto3==1.1.1
-
-RUN ln -s /usr/bin/nodejs /usr/bin/node
-
-RUN cd /opt/ && wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.6.4.tar.gz &&\
-    wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.6.4.tar.gz.asc &&\
-    wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl.asc &&\
-    gpg --import libressl.asc && gpg --verify libressl-2.6.4.tar.gz.asc libressl-2.6.4.tar.gz &&\
-    tar -xzf libressl-2.6.4.tar.gz && cd libressl-2.6.4 &&\
-    ./configure CFLAGS="-fPIC -O3" && make -j4 && make install &&\
-    cd /opt/ && rm -r libressl-2.6.4/ libressl-2.6.4.tar.gz libressl-2.6.4.tar.gz.asc libressl.asc
-
-RUN cd /opt && wget https://cmake.org/files/v3.12/cmake-3.12.1-Linux-x86_64.tar.gz -qO - | tar -xz
-
-RUN LANGUAGE=en_US.UTF-8 LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 locale-gen en_US.UTF-8
-
-RUN dpkg-reconfigure locales
+RUN pip install boto3==1.1.1 &&\
+	ln -s /usr/bin/nodejs /usr/bin/node &&\
+	cd /opt/ &&\
+	wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.6.4.tar.gz &&\
+	wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-2.6.4.tar.gz.asc &&\
+	wget https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl.asc &&\
+	gpg --import libressl.asc &&\
+	gpg --verify libressl-2.6.4.tar.gz.asc libressl-2.6.4.tar.gz &&\
+	tar -xzf libressl-2.6.4.tar.gz && cd libressl-2.6.4 &&\
+	./configure CFLAGS="-fPIC -O3" && make -j4 && make install &&\
+	cd /opt/ &&\
```

Comment:
This... seems a little bit overkill?  I understand not wastefully creating layers, but this would also mean updating cmake would require recompiling libressl, which sounds suboptimal.

Reply:
@alexmiller-apple Shouldn't be a problem unless we are updating cmake versions daily and not using a automated bot to build the image. I'll concede to creating separate layers if you have to perform the builds yourself.

Reply:
We do have to do the builds manually to test changes.  I don't get why we'd be doing work now that I'd have to undo the next time I want to add or change something in this file...

Reply:
I also still can't figure out what package you actually added